test/suites/integration/msp/stop-storing-bucket.test.ts

```diff
@@ -1,6 +1,5 @@
-import { strictEqual } from "node:assert";
+import assert, { strictEqual } from "node:assert";
 import { describeMspNet, mspKey, sleep, type EnrichedBspApi } from "../../../util";
-import invariant from "tiny-invariant";
```

Comment:
Much appreciated change! Should we also remove the dependency from `package.json`?

Reply:
Not yet. I think there is still some in the utils functions.